Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
We should make redis "optional" in the same way that mysql or postgresql is optional for activerecord. So a run-time requirement. But Action Cable ships with Rails by standard, not in the Gemfile.